Data/Unicode/Types.hs

```diff
@@ -95,4 +93,4 @@ data NormalizationMode
     | NFKD   -- ^ Compatibility decomposition.
     | NFC    -- ^ Canonical decomposition followed by canonical composition.
     | NFKC   -- ^ Compatibility decomposition followed by canonical composition.
-      deriving (Eq, Show, Enum, Typeable)
+      deriving (Eq, Show, Enum)
```

Comment:
We can add the following conditional for Typeable (and the same for the import above):

```
deriving (Eq, Show, Enum
#if __GLASGOW_HASKELL__ < 912
               , Typeable
```
               )

Reply:
Weird. I was sure I had tried that and couldn't get it working. This time? It just worked. Pushed.